ssh/tailssh/tailssh.go

```diff
@@ -1069,13 +1070,16 @@ func (ss *sshSession) run() {
 			ss.ctx.CloseWithError(err)
 		}
 	}()
+	var openOutputStreams atomic.Int32
+	openOutputStreams.Store(2)
```

Comment:
We only conditionally call `io.Copy` for stderr (it may be nil), so it seems like the starting value could be either 1 or 2.

Reply:
Done